gridt/src/app/core/settings.service.spec.ts

```diff
@@ -137,35 +137,30 @@ describe("IdentityService_AuthSuccesfull", () => {
   });
 
   // Local storage
-  it('should create localStorage upon first retreival of settings', () => {
+  it('should create localStorage upon first retrieval of settings', () => {
     // Subscribe local storage
-    service.the_user_settings$.subscribe();
+    service.the_user_settings$;
```

Comment:
This should theoretically do nothing, right?

Reply:
Good point, will fix in next commit.

Reply:
This does something: when I remove it, the test fails. However you are not subscribing you are running the 'get' function of this variable. Maybe it would be good to check here that this is equal to some expected value, especially when it has not been set before.